### ISSUE TITLE
can't download while providing download link

### ISSUE THREAD

**Issue by somewhat69** [chanderlud/giga-grabber#6]

**Comment by somewhat69** [comment]
when mega.nx link is given nothing happens

**Comment by chanderlud** [comment]
you may need to press enter in the text field or the "load all" button in order for the tool to load the provided link

**Comment by somewhat69** [comment]
the box turns red when i press enter or load all

**Comment by somewhat69** [comment]
also does it work on single files?

**Comment by chanderlud** [comment]
the red outline indicates that the tool was unable to load the URL, this can be for a variety of reasons. giga grabber only supports folders, but you can download only certain files from a folder if you want

**Comment by somewhat69** [comment]
no individual file with mega.nz/file can't be downloaded?

**Comment by chanderlud** [comment]
giga grabber only supports folders right now. if you want, you could contribute support for individual files

**Comment by somewhat69** [comment]
oh i see

**Comment by Manwithpants812** [comment]
Yeah single files is what people would want to use this for. Shame.

**Comment by chanderlud** [comment]
would it not be possible to put your single file into a folder and then download it that way?
